components/LeagueEntries/LeagueEntries.tsx

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+import { Button } from '../Button/Button';
+import { cn } from '@/utils/utils';
+import { EntryStatus } from '../EntryStatus/EntryStatus';
+import { ILeagueEntriesProps } from './LeagueEntries.interface';
+import * as React from 'react';
+
+const LeagueEntries = React.forwardRef<HTMLDivElement, ILeagueEntriesProps>(
+  ({ entryNumber, isEliminated = false, isPickSet = false }, ref) => (
```

Comment:
Simplify this. It will be easier for everyone else to read these types.

```suggestion
const LeagueEntries = (
  ({ entryNumber, isEliminated = false, isPickSet = false }: ILeagueEntriesProps, ref): HTMLDivElement => (
```

Reply:
Updated in latest commit.